frontend/benefit/handler/src/components/applicationReview/ApplicationReview.tsx

```diff
@@ -65,7 +65,7 @@ const ApplicationReview: React.FC = () => {
         <ConsentView data={application} />
         {application.status === APPLICATION_STATUSES.HANDLING && (
           <>
-            <SalaryBenefitCalculatorView />
+            <SalaryBenefitCalculatorView application={application} />
```

Comment:
lets rename param to data, to be consistent

Reply:
yes, this is resolved